Enterprise-CMCS/macpro-mdct-qmr
# Fixing QMR Favicon to be consistent with other MDCT Applications

### Description
In doing some debugging and general familiarization with the QMR application one of the first things I noticed was the Favicon was not that of MDCT but rather a previous contractor.  This PR aims to fix that by following patterns of MCR and MFP (see links below)

https://github.com/Enterprise-CMCS/macpro-mdct-mcr/blob/main/services/ui-src/public/index.html
https://github.com/Enterprise-CMCS/macpro-mdct-mfp/blob/main/services/ui-src/public/index.html


### Related ticket(s)
https://jiraent.cms.gov/browse/CMDCT-3090
---
### How to test
- Launch your browser in incognito mode
- Visit URL for this branch: https://dq9bz6nr9x1w7.cloudfront.net/MA/2023
- Note the Icon in the tab (also see image below) 
![qmr_favicon_fix](https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/52459927/9bebeebc-0114-4233-95bc-838e04649283)

### Important updates
N/A


---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
